src/component/tooltip/TooltipContent.js

```diff
@@ -250,11 +253,23 @@ TooltipContent.prototype = {
         if (domStyle.position !== 'absolute' && stl.position !== 'absolute') {
             domStyle.position = 'relative';
         }
+        this._moveTooltip()
         // Hide the tooltip
         // PENDING
         // this.hide();
     },
 
+    /**
+     * when alwaysShowContent is true change or rotation window size and restore will move tooltip
+     * @private
+     */
+    _moveTooltip: function() {
+        var ratio = this._styleCoord[2]; // The ratio of left to width
+        var realLeft = ratio * this._zr.getWidth();
+        var realRight = this._styleCoord[1];
+        this.moveTo(realLeft, realRight);
+    },
```

Comment:
The height ratio also should be handled.
```suggestion
    _moveTooltip: function() {
        var ratioX = this._styleCoord[2];
        var ratioY = this._styleCoord[3];
        var realX = ratioX * this._zr.getWidth();
        var realY = ratioY * this._zr.getHeight();
        this.moveTo(realX, realY);
    },
```